jujugui/static/gui/src/app/app.js

```diff
@@ -764,7 +768,12 @@ YUI.add('juju-gui', function(Y) {
         });
       });
 
-      controllerAPI.after('connectedChange', e => {
+      controllerAPI.after('connectedChange', evt => {
+        if (!evt.newVal) {
```

Comment:
I'm assuming that this is when the user logs out of the controller?
Could you add this example to the code comment plz.